### ISSUE TITLE
epub.js doesn't work with screen readers

### ISSUE THREAD

**Issue by sinabahram** [futurepress/epub.js#207]
When trying to use epub.js with a screen reader e.g. Jaws for Windows or the free and open source NVDA, the content is not readable by the screen reader in IE11 (this may just be an IE11 bug?).

Furthermore, in all browsers, the transport controls such as previous and next are divs instead of being semantic controls e.g. a <button>. If they must remain as divs, then we can fix this with role="button", but then keyboard functionality needs to be implemented instead of simply getting it for free by using appropriate html5 controls.

This library has limitless potential to open up the world of ebooks for the blind, and the first step to doing that is some simple html changes on the actual buttons and plugins. The next large step is figuring out why the content is inaccessible in IE11.

A fantastic feature would also be to have a button which would allow for the inlining of the entire book, sans any pagination, into the browser for blind users so that they can continuously read with a screen reader.

I'm more than willing to help.

**Comment by RawKStar77** [comment]
This is great, and something that's important for the future of this library. The controls do not have to remain as divs, and it would be great if you could use your knowledge to help make epub.js fully accessible. PRs are welcome!

> A fantastic feature would also be to have a button which would allow for the inlining of the entire book, sans any pagination, into the browser for blind users so that they can continuously read with a screen reader.

Pagination for a Chapter is done by CSS. The full text of the Chapter should be available to the screen reader. So the problem likely resides with problems with IE, which we have several open issues for:
+ https://github.com/futurepress/epub.js/issues/184
+ https://github.com/futurepress/epub.js/issues/192
+ https://github.com/futurepress/epub.js/issues/195
+ https://github.com/futurepress/epub.js/issues/150
+ https://github.com/futurepress/epub.js/issues/144
+ https://github.com/futurepress/epub.js/issues/50

**Comment by sinabahram** [comment]
Wow, thanks for the enthusiasm about accessibility. I really appreciate that! I’ll see about doing a first pass to get controls more accessible.

**Comment by eleanorz** [comment]
Unsubscribe

**Comment by JakeHartnell** [comment]
Closing for now, please checkout the new reader-repo: https://github.com/futurepress/epubjs-reader

Happy to review PRs for accessibility improvements.
